synapse/storage/schema/delta/32/reports.sql

```diff
@@ -15,6 +15,8 @@
 
 
 CREATE TABLE event_reports(
+    id BIGINT NOT NULL,
+    received_ts BIGINT NOT NULL,
```

Comment:
Maybe chuck an index on the received_ts? (Not the end of the world if you don't. I imagine it should be fairly easy to add one later)